### DISCUSSION TITLE
导入到 FlowUs

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#4772] [插件说明]
描述
---

> 将正文导入到 FlowUs，包括：正文、题图与 Favicon

地址
---

[https://simpread.ksria.cn/plugins/details/NgDkX2TqAx](https://simpread.ksria.cn/plugins/details/NgDkX2TqAx)

快捷键
---

`f s`

导入效果
---

![image](https://user-images.githubusercontent.com/81074/204078562-ddf8f206-7462-4c40-a375-011593942de9.png)

视频
---

[B 站视频](https://www.bilibili.com/video/BV1MP4y1Q7iE/) | [知乎视频](https://www.zhihu.com/zvideo/1586342219220520960)

https://user-images.githubusercontent.com/81074/204086212-4fcb541c-cad1-44da-9cd9-bae5b2fe991b.mp4

配置库
---

[简悦 · 配置库](https://github.com/Kenshin/simpread/discussions/4531) 是简悦官方推出的一套针对新用户的极简配置方案，方便新用户用最快的方式使用简悦的各种高级服务，配置库内置了常用的双链笔记用法，如：Notion、Obsidian、Logseq、Roam Research，同时包含了简悦在阅读模式上的一些常规插件：Live Editor、题图、Safari 阅读模式等。

目前配置库包含了 Obsidian 的一站式配置方案，细节 [请看这里](https://github.com/Kenshin/simpread/discussions/4531#discussioncomment-4407531)。

选项
---

随便进入一个页面 e.g. https://sspai.com/post/71576 并进入 **阅读模式 → 右下角 → 动作 → 插件触发器**

按照下图所示即可查看此插件的选项页。（注意：如果你的界面没有插件触发器，请根据此 [解决方案](https://github.com/Kenshin/simpread/discussions/2342) 修复。

<img width="1247" alt="image" src="https://user-images.githubusercontent.com/81074/204077978-a83f5ea8-cd3b-409f-930d-37c78a185363.png">

功能
---

1. 授权并获取 FlowUs 的空间与页面

   ![2022-11-26_15-45-10 (1)](https://user-images.githubusercontent.com/81074/204078228-bedd6758-02b1-4c7c-9752-55a623ad3c0e.gif)

2. 导入正文到 FlowUs

3. [导入题图到 FlowUs](https://github.com/Kenshin/simpread/discussions/4772#discussioncomment-4241217)

4. [导入 Favicon 到 FlowUs](https://github.com/Kenshin/simpread/discussions/4772#discussioncomment-4241523)

用法
---

1. [直接导入或加入稍后读时自动导入](https://github.com/Kenshin/simpread/discussions/4772#discussioncomment-4241531)

2. [在 Assistive Touch 设置 Icon 方便一键导出](https://github.com/Kenshin/simpread/discussions/4772#discussioncomment-4241578)

更新日志
---

### 1.0.2

1.  `Fixed` 非简悦高级账户权限使用 FlowUs 无法授权的 Bug。

### 1.0.1

1. `Fixed` 因为更换账户或授权的 Page 不存在后，无法重新授权的 Bug。

### 1.0.0

1. 授权并获取 FlowUs 的空间与页面

2. 导入正文到 FlowUs

3. [导入题图到 FlowUs](https://github.com/Kenshin/simpread/discussions/4772#discussioncomment-4241217)

4. [导入 Favicon 到 FlowUs](https://github.com/Kenshin/simpread/discussions/4772#discussioncomment-4241523)

引申
---

> 安装插件或带有插件触发器的插件失效的 [解决方案](https://github.com/Kenshin/simpread/discussions/2342)。

**Comment by Kenshin** [comment]
# 题图

支持题图，可获取当前页面的题图，依次优先级为：当前页面的默认题图 → [题图插件](https://github.com/Kenshin/simpread/discussions/2487) 设置的题图 → 当前页面的第一个图片。

当没有获取到题图或取消此功能（下图所示）时，不设定题图。（此选项在 **阅读模式 → 右下角 → 动作 → 插件触发器 → 导入到 FlowUs 选项**

<img width="556" alt="image" src="https://user-images.githubusercontent.com/81074/204078655-afb11da4-7ded-4bce-8933-462e839f5cb1.png">

## 效果

![image](https://user-images.githubusercontent.com/81074/204078562-ddf8f206-7462-4c40-a375-011593942de9.png)

**Comment by Kenshin** [comment]
# Favicon

支持 Favicon，可获取当前页面的 Favicon，Favicon 必须存在且是 `http` 开头才能生效。

当没有获取到题图或取消此功能（下图所示）时，不设定 Favicon。（此选项在 **阅读模式 → 右下角 → 动作 → 插件触发器 → 导入到 FlowUs 选项**

<img width="545" alt="image" src="https://user-images.githubusercontent.com/81074/204079009-97b4b34b-cfd7-4059-89c7-e2071c2ee590.png">

## 效果

![image](https://user-images.githubusercontent.com/81074/204078562-ddf8f206-7462-4c40-a375-011593942de9.png)

**Comment by Kenshin** [comment]
# 用法

## 直接导入

使用快捷键 `f s` 或 **阅读模式 → 右下角 → 动作 → 插件触发器 → 导入到 FlowUs**

https://user-images.githubusercontent.com/81074/204084469-a2f1edb6-c011-453a-92f7-84facd849a6d.mp4

## 加入稍后读后自动导入

通过设定 [自动化辅助增强](https://github.com/Kenshin/simpread/discussions/3596)，可实现当加入稍后读时，自动导入到 FlowUs。

按下图所示的位置加入 `f s` （如果下图所示的位置没有内容，直接加入，否则新起一行加入）

<img width="542" alt="image" src="https://user-images.githubusercontent.com/81074/204079454-67d5cf6b-8ca0-4e70-aa69-9184e113167a.png">

以下是导入效果

https://user-images.githubusercontent.com/81074/204084481-49f05dca-01d8-44aa-8f81-4e09ceff796a.mp4

**Comment by Kenshin** [comment]
# Assistive Touch

可使用 [Assistive Touch](https://github.com/Kenshin/simpread/discussions/1600) 实现下图效果。

<img width="1253" alt="image" src="https://user-images.githubusercontent.com/81074/204083834-ed23f141-a4d9-4987-b07a-8abb585725cc.png">

以下是导入效果

https://user-images.githubusercontent.com/81074/204084427-2542ab93-39b4-4bec-adec-c54c3bd4b723.mp4

**Comment by Salviaa37** [comment]
可否加入一个导入链接功能？

**Reply by guyeu9** [reply]
可以再加一个原文链接的功能，否则导入了有时候想看原文找不到

**Reply by Kenshin** [reply]
可以的，这两个是一个事情...

**Reply by guyeu9** [reply]
> 可以的，这两个是一个事情...

大佬，什么时候有时间更新一下啊，用这个比较多

**Reply by Kenshin** [reply]
预计下周更新 😂

**Reply by mynlbq** [reply]
导入到FLOWUS，还是没有原文链接？

**Reply by mynlbq** [reply]
插入原文链接这个功能，大佬，更新了吗

**Comment by Qwenjin** [comment]
请问下导出到flowus的时候，可以把直接图片上传到flowus服务器么，而不是仅图片链接复制。

**Reply by Kenshin** [reply]
下个版本支持。

**Reply by miigogo** [reply]
太需要了  坐等巨佬更新

**Reply by ggbondpx** [reply]
是已经支持这个功能了吗？我发现还是只有图片的链接

**Comment by luzov** [comment]
请问能不能支持二级页面的选择，目前好像只能选择到一级页面

**Reply by Kenshin** [reply]
会考虑的~

**Comment by yokiislucky** [comment]
你好，就是我用简悦插件阅读模式阅读文章，做了批注后导入flow us。但导入到flow us的文章没有批注，可不可以支持一下批注一起导入。

**Reply by Kenshin** [reply]
可以考虑，但标注只能是一个新 Page。
